examples/orientation_parser.html

```diff
@@ -0,0 +1,119 @@
+            // setting is "coordonnée geographiques en degres decimaux"
+            var positionOnGlobe = { longitude: 2.33481381638492, latitude: 48.850602961052147, altitude: 100 };
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate iTowns GlobeView
+            var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, {
+                handleCollision: false,
+            });
+           globeView.controls.minDistance = 30;
+
+            var menuGlobe = new GuiTools('menuDiv', globeView);
+
+            setupLoadingScreen(viewerDiv, globeView);
+
+            function addLayerCb(layer) {
+                return globeView.addLayer(layer);
+            }
+
+            // Add one imagery layer to the scene
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+
+            // Add elevation layer.
+            var promises = [];
+            promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
+            
+            // Input data : a geoJson file, with point features, with orientation specific properties
+            var panoramics = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/panoramicsMetaDataParis.geojson';
+            var layer = {
+                crsOut: globeView.referenceCrs,
+            };
+            // Input data are given in Lambert93 projection
+            itowns.proj4.defs('EPSG:2154', '+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+            // Add a promise to load oriented point datas
+            promises.push(
+                // first parse the JSON format
+                itowns.Fetcher.json(panoramics, {})
+                // then get geographic informations (coordinates) using GeoJson parser
+                .then(orientations => itowns.GeoJsonParser.parse(orientations, layer))
+                // then get orientation informations 
+                .then(collection => itowns.OrientationParser.parse(collection.features, layer))
+            );        
+            
+            // load collada model of a renault trafic
+            promises.push(ThreeLoader.load('Collada',
+             'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/collada/renault_trafic_no_texture.dae')
+             .then(collada => {
+                // prepare collada model base scale and rotation. in this example, we want the car to go straigth.
+                var colladaModel = collada.scene;
+                colladaModel.scale.set(0.5, 0.5, 0.5);
+                colladaModel.rotateZ(-Math.PI / 2 );
+                // put the model in a group, to allow user to do other transformation (rotation..)
+                var group = new itowns.THREE.Group();
```

Comment:
you can return directy `colladaModel`
and replace line 109 by
```js 
model.quaternion.premultiply(feature.quaternion);
```

Reply:
if you want, I didn't see previous [comment ](https://github.com/iTowns/itowns/pull/830#discussion_r209332056)